bsconfig.json

```diff
@@ -0,0 +1,13 @@
+/* bsconfig.json */
+{
+  "name": "wp-calypso",
+  "sources": [
+    "client/bs"
+  ],
+  "bs-dependencies": [
+    "reason-react"
+  ],
+  "reason": {
+    "react-jsx": true
```

Comment:
https://github.com/rrdelaney/bs-loader

but you can see that @rrdelaney updated that in https://github.com/rrdelaney/bs-loader/commit/9134552830723ff313a988d32e70d15db5fe1bd8 recently